redbot/cogs/trivia/data/lists/minecraft.yml

```diff
@@ -0,0 +1,114 @@
+AUTHOR: ImajinDevon
+
+"The phantom was added in which update?":
+ - "1.13"
+"Which update introduced a new style combat into the game?":
+ - "1.9"
+'What edition of Minecraft is usually prefixed with the word "bug"?':
+ - "Bedrock"
+"Who created Minecraft?":
+ - "Notch"
+"Who bought Mojang in 2014?":
+ - "Microsoft"
+"Mojang was bought for (...) billion dollars:":
+ - "2.5"
+"If you name a sheep this, it will turn rainbow:":
+ - "jeb_"
+"What supercharges creepers?":
+ - "lightning"
+"What are creepers scared of?":
+ - "cats"
+"To kill an iron golem without getting hit, you must build how many blocks up?":
+ - "3"
+"What is an enderman's weakness?":
+ - "water"
+"What block lights up when you step on it?":
+ - "redstone ore"
+"If you trip this, redstone power will be emitted:":
+ - "tripwire hook"
+"Upon opening this container, redstone power will be emitted:":
+ - "trap chest"
+"This myth has commonly passed around the Minecraft community for years:":
+ - "herobrine"
+"To obtain the dragon's egg, you must have it fall onto which item?":
+ - "torch"
+"What mob dances when music played?":
+ - "parrot"
+ - "parrots"
+"What is the name of the most popular arcade and minigame server?":
+ - "hypixel"
+"What is the default port of a Minecraft server?":
+ - "25565"
```

Comment:
There is 3 different default ports, default for java is 25565 while bedrock's default is 19132 on IPv4 while IPv6 uses 19133 by default.